### PR TITLE
Add form

### PR DESCRIPTION
**Pull request by abreham1222**
This part of the contact form the figma design for the mobile version:
--we created this pr:
1.  Added index.html
2. Added style.css
3. Added all icons

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -215,6 +215,42 @@ <h5 class="text-white">Skills</h5>
               </ul>
             </div>
         </section>
+      <form action="https://formspree.io/f/xlezlkyw" method="post">
+        <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+        <input name="full-name" type="text" placeholder="Full name" maxlength="30" required>
+        <input name="email" type="email" placeholder="Email address" required>
+        <textarea name="message" id="message" cols="30" rows="10" placeholder="Enter text here"></textarea>
```
Comment (omar-muhamad) [inline index.html]: - [x] As required a user cannot submit the form if one or more of the fields is empty. So please add a `required` attribute as you did in the other fields.
- [x] Please make the message text area field accepts up to 500 chars.

image

File: index.html [hunk 2]
```diff
@@ -215,6 +215,42 @@ <h5 class="text-white">Skills</h5>
...
+              <img src="Images/ico/twit.png" alt="Twitter icon for check"/>
+            </a>
+          </li>
+          <li>
+            <a href="#" class="social-media-icons">
+              <img src="Images/ico/med.png" alt="medium icon for check"/>
+            </a>
+          </li>
+          </ul>
+          <img class="shape" src="Images/ico/Shape.png" alt="shape">
```
Comment (omar-muhamad) [inline index.html:255]: - [x] For semantics purposes Please remove these social links list from the `<form>` tag and add it to a `<footer>` tag.

File: index.html (hunk 1 above)
Comment (omar-muhamad) [inline index.html]: - [x] Please add the underline below the name and email input fields and the background color for the text area to make your design match the given design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| image  | image |

File: style.css [hunk 3]
```diff
@@ -322,3 +322,87 @@ a {
...
+  font-size: 17px;
+  line-height: 24px;
+  background: #f55800;
+  color: #fff;
+  margin: 24px 122px;
+  align-items: center;
+  padding: 12px;
+  width: 131px;
+  height: 48px;
+}
```
Comment (omar-muhamad) [inline style.css:386]: - [ ] Please add `hover` and `pressed` button interaction styles by adding `hover` and `active` pseudo classes with the given Figma design.

image

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @abreham1222,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by MahmoudBakr23 — APPROVED** [review]
Hi team 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done making the changes requested by the previous reviewer.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**